docs/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+**Description of changes**
```

Comment:
I think this should be in `.github/pull_request_template.md`.
However, this will be the one set for all PRs. 

I think this is a way to have multiple templates:
https://stackoverflow.com/a/75030350